README.md

```diff
@@ -105,7 +105,7 @@ The content of this library is released under the **MIT License** by
 **Andrew Lawson**.<br/> You can find a copy of this license in
 [`LICENSE`][license] or at http://opensource.org/licenses/mit.
 
-[boxes]:       https://vagrantcloud.com
+[boxes]:       https://atlas.hashicorp.com
```

Comment:
This url seems to point to a login page, maybe [https://atlas.hashicorp.com/boxes/search](https://atlas.hashicorp.com/boxes/search) would be better?